lego-webapp/components/MazemapEmbed/MazemapAccordion.tsx

```diff
@@ -14,7 +15,9 @@ export const MazemapAccordion = ({ defaultOpen = false, ...props }: Props) => {
   return (
     <>
       <Flex column className={styles.mazemapAccordion}>
+        <Helmet title="Mazemap">{mazemapScript}</Helmet>
```

Comment:
```suggestion
        <Helmet>{mazemapScript}</Helmet>
```
This sets the title of all meetings and events using mazemap to "Mazemap"
<img width="221" alt="Screenshot 2025-03-14 at 19 35 17" src="https://github.com/user-attachments/assets/050104f2-f8d0-41c3-b036-af66048f1740" />

Reply:
Same for "Mazemap Search"